### PR TITLE
FOSS4G Workshops preperation

### PR DESCRIPTION
**Pull request by ianturton**

### CODE REVIEW

File: docs/src/main/resources/org/geotools/tutorial/csv/locations.csv [hunk 1]
```diff
@@ -13,3 +13,4 @@ LAT, LON, CITY, NUMBER, YEAR
 52.95, -1.133333, Nottingham, 800, 2013
 45.52, -122.681944, Portland, 840, 2014
 37.5667,129.681944,Seoul,473,2015
+50.733992,7.099814,Bonn,700,2016
```
Comment (bradh) [inline docs/src/main/resources/org/geotools/tutorial/csv/locations.csv]: Newline at end?

File: docs/user/tutorial/datastore/artifacts/locations.csv [hunk 2]
```diff
@@ -13,3 +13,4 @@ LAT, LON, CITY, NUMBER, YEAR
 52.95, -1.133333, Nottingham, 800, 2013
 45.52, -122.681944, Portland, 840, 2014
 37.5667,129.681944,Seoul,473,2015
+50.733992,7.099814,Bonn,700,2016
```
Comment (bradh) [inline docs/user/tutorial/datastore/artifacts/locations.csv]: Newline at end?

File: build/maven/pom.xml [hunk 3]
```diff
@@ -16,7 +16,7 @@
   <parent>
     <groupId>org.geotools.maven</groupId>
     <artifactId>build</artifactId>
-    <version>17-SNAPSHOT</version>
+    <version>17.2</version>
```
Comment (bradh) [inline build/maven/pom.xml]: Do you really want to merge that into 17.x branch?

File: README.html [hunk 4]
```diff
@@ -23,9 +23,9 @@
 </HEAD>
 <BODY LANG="en-AU" DIR="LTR">
 <a href="http://geotools.org"><img src="README-geotools-logo.png"></a>
-<H1 CLASS="western">GeoTools @RELEASE@</H1>
-<P>Thank you for downloading GeoTools @RELEASE@. This release was
-created on @DATE@.</P>
+<H1 CLASS="western">GeoTools 17.2</H1>
+<P>Thank you for downloading GeoTools 17.2. This release was
+created on Jul 31, 2017.</P>
```
Comment (bradh) [inline README.html]: Do you really want this in 17.x branch?

File: docs/user/tutorial/datastore/artifacts/locations.csv (hunk 2 above)
Comment (ianturton) [inline docs/user/tutorial/datastore/artifacts/locations.csv]: yep

File: docs/src/main/resources/org/geotools/tutorial/csv/locations.csv (hunk 1 above)
Comment (ianturton) [inline docs/src/main/resources/org/geotools/tutorial/csv/locations.csv]: yep

**Comment by ianturton** [comment]
need to branch off 17.x not 17.2!

**Pull request closed without merging**